src/lib/components/PriceTag.svelte

```diff
@@ -21,6 +21,11 @@
 	 * Convert to the secondary currency
 	 */
 	export let secondary = false;
+	/**
+	 * Optional precision for displaying the amount. If provided, uses this precision
+	 * instead of the default FRACTION_DIGITS_PER_CURRENCY.
+	 */
+	export let precision: number | undefined = undefined;
```

Comment:
As far as I can tell, this is used nowhere.

Why do we support `precision` to be undefined? Wouldn't it be simpler to support a single form of price? It would certainly eliminate this issue.